src/pspm_convert_ppg2hb.m

```diff
@@ -81,96 +85,152 @@
 %% Large spikes mode
 %--------------------------------------------------------------------------
```

Comment:
The header "Large spikes mode" is placed wrongly (different from PR #612) - it should precede the associated code.

Reply:
<img width="1096" alt="image" src="https://github.com/bachlab/PsPM/assets/3895146/9e2f016f-6965-4d77-8bde-84a3b2976949">
Sorry I am just a little unsure. I just noticed in the screenshot above (in PR #613 ) `%% Large spikes mode` has been removed. Shall I remove it here too? I think in the version you reviewed, I just forgot to remove this line..

Reply:
Yes, please remove this line here.